libraries/commerce/product/reducers/productRelations.js

```diff
@@ -0,0 +1,48 @@
+import {
+  REQUEST_PRODUCT_RELATIONS,
+  RECEIVE_PRODUCT_RELATIONS,
+  ERROR_PRODUCT_RELATIONS,
+  PRODUCT_LIFETIME,
+} from '../constants';
+
+/**
+ * Product relations reducer.
+ * @param {Object} state State.
+ * @param {Object} action Action.
+ * @returns {Object}
+ */
+const productRelations = (state = {}, action) => {
```

Comment:
To keep the naming pattern of the other product reducers i think that this should be called `productRelationsByHash`.

Reply:
Done.